androidApp/src/main/java/com/EENX15_22_17/digital_journal/android/screens/current/currentpatients/CreatePatientDialog.kt

```diff
@@ -0,0 +1,136 @@
+package com.EENX15_22_17.digital_journal.android.screens.current.currentpatients
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.*
+import androidx.compose.runtime.*
+import androidx.compose.runtime.saveable.rememberSaveable
+import androidx.compose.ui.ExperimentalComposeUiApi
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.window.Dialog
+import androidx.compose.ui.window.DialogProperties
+import com.EENX15_22_17.digital_journal.android.ui.components.TitledTextField
+import com.EENX15_22_17.digital_journal.android.ui.components.TitledDateField
+import com.EENX15_22_17.digital_journal.android.ui.components.TitledTextFieldDigitKeyboard
+import com.EENX15_22_17.digital_journal.android.ui.components.TitledTimeField
+import java.time.LocalDate
```

Comment:
These are the Java time-types (which historically have been quite troublesome/ambiguous to work with), the new `Local*`-types are better in this regard, but not supported everywhere (which is why you get the API warning).

I would suggest using https://github.com/Kotlin/kotlinx-datetime instead, however, there is no release which includes equvalent types to javas `LocalDate` and `LocalTime`. Instead we would need to use `LocalDateTime`, and change portions of this value, so a bit more troublesome.
`kotlinx.datetime` is also a bit more explicit to work with, so you would have to create the `LocalDateTime` from an `Instant` with a `TimeZone`. I would propose a utility extension-function for this:
```kotlin
fun Clock.localDateTimeAtCurrentTimeZone(): LocalDateTime {
    val instant = now()
    val currentTimeZone = TimeZone.currentSystemDefault()
    return instant.toLocalDateTime(currentTimeZone)
}
```